docs/data_schema.md

```diff
@@ -143,16 +145,13 @@ The top-level identifying keys have been revised a little bit from the dynamic a
     "files": [
       {
         "filename": string,
-        "basic": {
-          "description": string,
-          "size": int,
-          "sha256": string,
-          "line_lengths": [
-            { "value": int, "count": int }
-          ]
-        },
-        "parsing": {
-          "language": string,
+	    "detected_type": string,
```

Comment:
nit: the indents are a bit inconsistent.

Reply:
fixed, had accidentally inserted tabs